suite/volume/flux_volume_test.go

```diff
@@ -0,0 +1,86 @@
+package volume
+
+import (
+	"context"
+	"github.com/avast/retry-go"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/smartcontractkit/integrations-framework/client"
+	"github.com/smartcontractkit/integrations-framework/contracts"
+	"github.com/smartcontractkit/integrations-framework/tools"
+	"math/big"
+	"time"
+)
+
+var _ = Describe("Flux monitor volume tests", func() {
+	jobPrefix := "flux_monitor"
+	spec := &FluxTestSpec{
+		TestSpec: TestSpec{
+			InitFunc:                client.NewHardhatNetwork,
+			OnChainCheckAttemptsOpt: retry.Attempts(120),
+		},
+		AggregatorsNum:      20,
+		RequiredSubmissions: 5,
+		RestartDelayRounds:  0,
+		JobPrefix:           jobPrefix,
+		NodePollTimePeriod:  15 * time.Second,
+		FluxOptions:         contracts.DefaultFluxAggregatorOptions(),
+	}
+	ft := NewFluxTest(spec)
+	Describe("round completion times", func() {
+		promRoundTimeout := 120 * time.Second
+		currentRound := 1
+		rounds := 5
+		// just wait for the first round to settle about initial data
+		err := ft.awaitRoundFinishedOnChain(currentRound, tools.VariableData)
+		Expect(err).ShouldNot(HaveOccurred())
+		_, err = ft.roundsStartTimes()
+		Expect(err).ShouldNot(HaveOccurred())
+
+		Measure("Should process rounds without errors", func(b Benchmarker) {
+			blockStart, err := ft.DefaultSetup.Client.BlockNumber(context.Background())
+			Expect(err).ShouldNot(HaveOccurred())
+			// all contracts/nodes/jobs are setup at that point, triggering new round,
+			// waiting for all contracts to complete one round and get metrics from runs/on-chain
+			newVal, err := ft.Adapter.TriggerValueChange(currentRound)
+			Expect(err).ShouldNot(HaveOccurred())
+
+			ctx, cancel := context.WithTimeout(context.Background(), promRoundTimeout)
+			defer cancel()
+
+			// await all nodes report round was finished
+			roundFinished, err := ft.Prom.AwaitRoundFinishedAcrossNodes(ctx, currentRound+1)
```

Comment:
We're also checking on-chain to see if the rounds completed, I wouldn't have both a prometheus & on-chain check. On chain is what we should solely depend on.